test/e2e/resource_quota.go

```diff
@@ -451,6 +522,55 @@ func newTestPodForQuota(name string, requests api.ResourceList, limits api.Resou
 	}
 }
 
+// newTestPersistentVolumeClaimForQuota returns a simple persistent volume claim
+func newTestPersistentVolumeClaimForQuota(name string) *api.PersistentVolumeClaim {
+	return &api.PersistentVolumeClaim{
+		ObjectMeta: api.ObjectMeta{
+			Name: name,
+		},
+		Spec: api.PersistentVolumeClaimSpec{
+			AccessModes: []api.PersistentVolumeAccessMode{
+				api.ReadWriteOnce,
+				api.ReadOnlyMany,
+				api.ReadWriteMany,
+			},
+			Resources: api.ResourceRequirements{
+				Requests: api.ResourceList{
+					api.ResourceName(api.ResourceStorage): resource.MustParse("1Gi"),
+				},
+			},
+		},
+	}
+}
+
+// newTestReplicationControllerForQuota returns a simple replication controller
+func newTestReplicationControllerForQuota(name string, image string, replicas int) *api.ReplicationController {
```

Comment:
nit, I am a huge fan of `(name, image string, replicas int)` over `(name string, image string, replicas int)`